### DISCUSSION TITLE
Questions before purchasing copilot

### DISCUSSION THREAD

**Discussion by cafe24-jkjang** [community/community#42743] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, my team has 20 developers that wants to use copilot and I would like to check a few things before buying & using

1. Is there a way to buy several copies of copilot for individuals at once?
I know there is copilot for business, but our team doesn't use github enterprise so we can't match the price
I created a organization in github and tried to buy several copies in the marketplace but copilot seems to not be available 
 in the marketplace anymore

2. After purchasing copilot for individuals, can I use my github account for copilot on several devices at the same time?
I checked this discussion but I just wanted to double check if it still works https://github.com/community/community/discussions/42009

Thank you!

**Comment by cafe24-jkjang** [comment]
One more!

3. Is it ok for a company to use github copilot for individuals? Thank you!

**Comment by cmp-nct** [comment, marked as answer]
I'd recommend reading the terms of service.
I can not recall finding any such restrictions (as they have barely any terms), obviously they do not want you to do it and they might rate limit you in some way but you can share github account(s), especially when behind the same IP address that's likely going to work.

My guess is that their devs are completely occupied getting GPT 3.5 to work, Copilot will die (become obsolete) within a year if they can't match the quality we see there. That's why the progress on such topics is so slow.

**Reply by cafe24-jkjang** [reply]
Thank you for sharing this :)

**Comment by datanerd** [comment]
> Is there a way to buy several copies of copilot for individuals at once?
> I know there is copilot for business, but our team doesn't use github enterprise so we can't match the price I created a organization in github and tried to buy several copies in the marketplace but copilot seems to not be available in the marketplace anymore

GitHub Copilot for Individual license is linked with your GutHub account, you can't associate multiple Copilot for Individual licenses with a single GitHub username. If you are thinking about getting everyone in your team a Copilot license, each individual will need to get it themselves.
